File: 283-move-zeroes/283-move-zeroes.py
class Solution:
    def moveZeroes(self, nums: List[int]) -> None:
        """
        Do not return anything, modify nums in-place instead.
        """
        n = len(nums)
        counter = 0
        j= 0
        for i in nums:
            if i != 0:
                counter+=1
        for i in range(n):
            if j < counter:
                if nums[i] !=0:
                    nums[j] = nums[i]
                    j+=1
        for i in range(j,n):
            nums[i] = 0

File: 283-move-zeroes/283-move-zeroes.cpp
class Solution {
public:
    void moveZeroes(vector<int>& nums) {
        int count = 0;
        int temp = 0;
        for(int i = 0;i<nums.size();i++)
        {
            if(nums[i]!=0)
            {
                temp = nums[i];
                nums[i]= nums[count];
                nums[count] = temp;
                count++;
            }
        }
    }
};